common/shlibs

```diff
@@ -974,6 +974,7 @@ libfftw3f_threads.so.3 libfftw-3.3_1
 libfftw3f.so.3 libfftw-3.3_1
 libfftw3f_omp.so.3 libfftw-3.3.5_1
 libfluidsynth.so.1 libfluidsynth-1.1.5_1
+libinstpatch-1.0.so.2 libinstpatch-1.1.4_1
```

Comment:
Is this something other packages link against? If not, it doesn't need to be here.

Reply:
I plan on packaging software with a dependency, should changing shlibs be done in the commit that adds the package using it?